src/engine.c

```diff
@@ -34,13 +34,48 @@ char *exp_names[MAX_EXPANSION + 1] =
 	"The Brink of War",
 	"Alien Artifacts",
 	"Xeno Invasion",
+	"Rebel vs Imperium (without Gathering Storm)",
```

Comment:
Yes, just a minor quibble there, I think it will be obvious. Go with "Rebel vs Imperium (only)".